### PR TITLE
remove stargate parent pom

### PR DESCRIPTION
**Pull request by Yuqi-Du**
(https://github.com/stargate/data-api/issues/1341)

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,42 +1,57 @@
...
     <stargate.int-test.coordinator.image>stargateio/coordinator-dse-next</stargate.int-test.coordinator.image>
-    <stargate.int-test.coordinator.image-tag>v${stargate.version}</stargate.int-test.coordinator.image-tag>
+    <stargate.int-test.coordinator.image-tag>v2.1.0-BETA-14</stargate.int-test.coordinator.image-tag>
     <stargate.int-test.cluster.name>dse-next-${stargate.int-test.cassandra.image-tag}-cluster</stargate.int-test.cluster.name>
     <stargate.int-test.cluster.persistence>persistence-dse-next</stargate.int-test.cluster.persistence>
     <stargate.int-test.cluster.dse>false</stargate.int-test.cluster.dse>
     <stargate.int-test.cluster.hcd>false</stargate.int-test.cluster.hcd>
     <stargate.int-test.cassandra.auth-enabled>true</stargate.int-test.cassandra.auth-enabled>
     <stargate.int-test.use-coordinator>true</stargate.int-test.use-coordinator>
+    <!-- driver -->
```
Comment (tatu-at-datastax) [inline pom.xml:37]: nit: driver -> Cassandra driver ?

File: pom.xml [hunk 2]
```diff
@@ -243,6 +259,45 @@
...
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${surefire-plugin.version}</version>
+        <configuration>
+          <argLine>-Xmx4g</argLine>
+          <systemPropertyVariables>
+            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+            <maven.home>${maven.home}</maven.home>
+          </systemPropertyVariables>
+          <excludes>
+            <exclude>**/*EndToEndTest.*</exclude>
```
Comment (tatu-at-datastax) [inline pom.xml:281]: Can remove, I think?

**Review by tatu-at-datastax — APPROVED** [review]
LGTM

File: pom.xml (hunk 1 above)
Comment (Yuqi-Du) [reply pom.xml:37]: solved, thanks

File: pom.xml (hunk 2 above)
Comment (Yuqi-Du) [reply pom.xml:281]: solved, thanks

**Review by tatu-at-datastax — APPROVED** [review]

**Pull request merged**